example/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  devtool: 'cheap-module-eval-source-map',
+  entry: [
+    'webpack-hot-middleware/client',
+    './index'
+  ],
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js',
+    publicPath: '/static/'
+  },
+  resolve: {
+    alias: {
+      'react-maxwell-modal': path.join(__dirname, '..','src', 'index.js')
+    }
+  },
+  plugins: [
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoErrorsPlugin()
+  ],
+  module: {
+    loaders: [{
+      test: /\.jsx?$/,
+      loaders: ['babel'],
```

Comment:
@securingsincity you should be able to move transform logic from the .babelrc into this loader